### ISSUE TITLE
Audio and Video out of Sync 

### ISSUE THREAD

**Issue by AndreaMonzini** [mltframework/mlt#240]
Hello,
i'm testing Linux alternative for video editing and i have problems with audio out of sync in playback with apps using MLT.

As soon as i move the playhead or jump the play in different position the audio shift out ot sync ( no clip editing ).
The projec is set to 25 fps as the video ( constant frame rate ).

Kdenlive 17.04.1 MLT 6.5.0 ( Audio out of Sync )
Shotcut 17.06.01 MLT ?        ( Audio out of Sync )

Kdenlive 16.12.2 MLT 6.4.1 ( Audio in Sync, no problems)

Linux kernel 4.9.0 
GNOME Shell 3.22.3

Video data from Mediainfo:

Video
ID                                       : 1
Format                                   : VP9
Codec ID                                 : V_VP9
Duration                                 : 3 min 1 s
Width                                    : 1 920 pixels
Height                                   : 1 080 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 25.000 FPS
Language                                 : English
Default                                  : Yes
Forced                                   : No

Audio
ID                                       : 2
Format                                   : Opus
Codec ID                                 : A_OPUS
Duration                                 : 3 min 1 s
Channel(s)                               : 2 channels
Channel positions                        : Front: L R
Sampling rate                            : 48.0 kHz
Compression mode                         : Lossy
Language                                 : English
Default                                  : Yes
Forced                                   : No

**Comment by ddennedy** [comment]
OK, I reproduced this in Shotcut 17.06 by using the Count down 2-pop generator and exporting it as VP9 WebM and then replaying it. MLT 6.5.0 is an unreleased version, but Shotcut uses MLT git head. By using the monthly releases I was able to narrow down the regression to the 17.03 release. More than likely, the problem came with a commit in the avformat producer in February, and we can see there were some interesting commits in that month:
https://github.com/mltframework/mlt/commits/master/src/modules/avformat/producer_avformat.c

This problem appears to be limited to codec or format as I do not reproduce it with H.264+AAC MP4, VP8+Vorbis WebM, ProRes+PCM mov, DNxHD+PCM mov.
